### DISCUSSION TITLE
KEDA Chart v2.8.0

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/charts#299] [Announcements]
*Chart: v2.8.0 | App: v2.8.0*

## TL;DR
```shell
helm repo add keda https://kedacore.github.io/charts
helm install keda/keda
```

### Features

- Support for KEDA v2.8.0 ([release notes](https://github.com/kedacore/keda/releases/tag/v2.8.0))
- Support for controlling replica count of metric server* (https://github.com/kedacore/charts/pull/295)
- Support for controlling timestamp format in KEDA operator (https://github.com/kedacore/charts/pull/278)
- Support for relabelling metrics in Prometheus  (https://github.com/kedacore/charts/pull/287 - Contributed by @bmiguel-teixeira 🎉)
- Support for defining affinity for Metrics API and/or operator (https://github.com/kedacore/charts/pull/294 - Contributed by @aslafy-z 🎉)
- Support for defining update strategy for Metrics API and/or operator (https://github.com/kedacore/charts/pull/292 - Contributed by @aslafy-z 🎉)
- Support for using `AutomountServiceAccountToken`  (https://github.com/kedacore/charts/pull/258- Contributed by @ph311o  🎉)

> ⚠️ While you can run more replicas of our metric server, only one instance will used and serve traffic.
> 
> You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce downtime during a failover.

### Fixes / Changes

- Improve Prometheus workload labelling to only be used when pod monitor is not enabled (https://github.com/kedacore/charts/pull/279 - Contributed by @mhowell24 🎉)

### Breaking Changes

None.

### Removal

None.

## New Contributors
* @Swalloow made their first contribution in https://github.com/kedacore/charts/pull/276
* @mhowell24 made their first contribution in https://github.com/kedacore/charts/pull/279
* @ph311o made their first contribution in https://github.com/kedacore/charts/pull/258
* @bmiguel-teixeira made their first contribution in https://github.com/kedacore/charts/pull/287
* @aslafy-z made their first contribution in https://github.com/kedacore/charts/pull/292

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.7.2...v2.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.8.0'>KEDA Chart v2.8.0</a>.</em>
